lib/taskana-core-test/src/test/java/acceptance/user/UserServiceAccTest.java

```diff
@@ -913,6 +1042,25 @@ void should_ReturnOneDomain_When_GroupHasSufficientMinimalPermissionsToAssignDom
       assertThat(userInDatabase.getDomains()).containsExactly(workbasket.getDomain());
     }
 
+    @WithAccessId(user = "user-1-1")
+    @Test
+    void should_ReturnOneDomain_When_PermissionHasSufficientMinimalPermissionsToAssignDomains()
```

Comment:
Why is this test nessecary in addition to should_DetermineDomains_When_WorkbasketPermissionsExistForUsersWithPermissions() ?

Reply:
really don't know it myself 🤡 I'll probably just remove it